### DISCUSSION TITLE
Add batch file support to match the error message

### DISCUSSION THREAD

**Discussion by hello-smile6** [DustinBrett/daedalOS#63] [Ideas]
Why reference batch files if they're unsupported?

https://github.com/DustinBrett/daedalOS/blob/b7a6d160150885a9a2eb464afdb9b32ca4aa6cec/components/apps/Terminal/functions.ts#L111

**Comment by DustinBrett** [comment]
That response is to emulate the exact way it's done in cmd.exe. Python files are supported. I will add batch to my to do list cause I'd like to keep the message as is.

**Reply by hello-smile6** [reply]
Okay, why not match a different OS?

**Reply by DustinBrett** [reply]
Cause I use Windows 10 and I liked it.

**Reply by hello-smile6** [reply]
Oh.

**Reply by DustinBrett** [reply]
The original point of this app and my site was to recerate the experience of someone sitting at my PC (which runs Windows 10). So that is where this began. For that reason, I want to keep the message as is but I do indeed agree it should be accurate so for me the goal would be to add `.bat` files, which shouldn't be too hard.

**Reply by hello-smile6** [reply]
Oh. Wouldn't it basically just have to be bash, but with some features removed and some bugs added?

**Reply by DustinBrett** [reply]
Not really imo. The point is to emulate Windows 10 behaviors.
